auxx/src/Lang/Name.hs

```diff
@@ -27,11 +30,17 @@ instance Arbitrary Letter where
     arbitrary = Letter <$> arbitrary `suchThat` isAlpha
 
 newtype Name = Name (NonEmpty (NonEmpty Letter))
-    deriving (Eq, Ord, Generic)
+    deriving stock   (Eq, Ord)
+    deriving newtype (Generic)
```

Comment:
Why did you do this change?

Reply:
I just tried to make `Arbitrary` instance compilable. Probably this change is non needed, will revert it back.